editor.html

```diff
@@ -70,7 +70,7 @@
     <body>
         <script type="application/javascript">
             // VERSION INFORMATION
-            EDITOR_VERSION = "1.1.2";
+            EDITOR_VERSION = "1.1.2-webusb";
```

Comment:
Same as the changelog comment, we should probably edit the version only in the integration branches (master or beta).